AbletonMCP_Remote_Script/handlers/audio.py

```
@@ -0,0 +1,255 @@
+"""Audio: load samples, warp, crop, reverse, analyze, freeze, export."""
+
+from __future__ import absolute_import, print_function, unicode_literals
+
+import traceback
+
+
+def get_audio_clip_info(song, track_index, clip_index, ctrl=None):
+    """Get information about an audio clip."""
+    try:
+        if track_index < 0 or track_index >= len(song.tracks):
+            raise IndexError("Track index out of range")
```

Ruff (0.14.14)

[warning] Abstract `raise` to an inner function

(TRY301)